src/lib/list/selection-list.spec.ts

```diff
@@ -123,6 +123,31 @@ describe('MatSelectionList without forms', () => {
       expect(listOptions[2].nativeElement.getAttribute('aria-disabled')).toBe('false');
     });
 
+    it('should be able to specify a color for list options', () => {
+      expect(listOptions.map(option => option.nativeElement)
+          .every(option => !option.classList.contains('mat-primary'))).toBe(true);
+      expect(listOptions.map(option => option.nativeElement)
+        .every(option => !option.classList.contains('mat-warn'))).toBe(true);
+
+      // All options will be set to the "warn" color.
+      fixture.componentInstance.selectionListColor = 'warn';
+      fixture.detectChanges();
+
+      expect(listOptions.map(option => option.nativeElement)
+        .every(option => !option.classList.contains('mat-primary'))).toBe(true);
+      expect(listOptions.map(option => option.nativeElement)
+        .every(option => option.classList.contains('mat-warn'))).toBe(true);
+
+      // Color will be set explicitly for an option and should take precedence.
+      fixture.componentInstance.firstOptionColor = 'primary';
+      fixture.detectChanges();
+
+      expect(listOptions[0].nativeElement.classList).toContain('mat-primary');
```

Comment:
Consider mapping the `listOptions` to their native elements at the top of the test so you don't have to repeat the `.nativeElement` everywhere.

Reply:
Done. good idea. thanks